doc/quickstart/app.md

```diff
@@ -0,0 +1,26 @@
+# Deploy an app
+
+## 1. Click on "Dashboards" -> "New"
+
+![](../static/dashboards-new.png)
+
+
+## 2. Fill out the form
+
+
+![](../static/dashboards-form.png)
```

Comment:
This image displays `Ensure Voila is selected`. I think user should be able to select any of the options?

Reply:
streamlit and shiny don't work (they display a banner saying "streamlit/shiny is coming soon"). 

for docker, we don't have many examples yet, so voila is the one we chose for showing how to deploy something quickly